.github/workflows/cleanup-pre-releases.yaml

```
@@ -12,18 +12,28 @@ permissions:
     pull-requests: read
 
 jobs:
-    cleanup-prereleases:
+    cleanup-pre-releases:
         runs-on: ubuntu-latest
         steps:
             - name: Checkout repository
               uses: actions/checkout@v3
```

actionlint (1.7.7)

the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)